pkgs/applications/graphics/kcc/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "KindleComicConverter";
+  version = "5.5.1";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "5dbee5dc5ee06a07316ae5ebaf21ffa1970094dbae5985ad735e2807ef112644";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ pillow pyqt5 psutil python-slugify raven ];
+
+  meta = {
+    description = "Python app to convert comic/manga files or folders to EPUB, Panel View MOBI or E-Ink optimized CBZ.";
+    homepage = https://kcc.iosphe.re;
```

We're likely going to deprecate unquoted URL syntax soon, when RFC 45 is approved.

```suggestion
    homepage = "https://kcc.iosphe.re";
```